win32ss/user/user32/controls/edit.c

```diff
@@ -3981,6 +3992,35 @@ static void EDIT_WM_SetFocus(EDITSTATE *es)
 	notify_parent(es, EN_SETFOCUS);
 }
 
+static DWORD get_font_margins(HDC hdc, const TEXTMETRICW *tm, BOOL unicode)
+{
+	ABC abc[256];
```

Comment:
They use tabs here in wine?

Reply:
yes but I don’t think that affects the WINESYNC script much so let’s convert them